email_address.gemspec

```diff
@@ -32,5 +32,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "simplecov"
 
   spec.add_dependency "simpleidn"
-  spec.add_dependency "netaddr", "~> 2.0"
+  spec.add_dependency "netaddr", "~> 2.0.4"
```

Comment:
It another lock. In the current version `2.1` is allowable, in suggested — no.

So, `'>= 2.0.4', '< 3'` is better.

Reply:
Agreed, I've pushed the change.